backend/storage/storage-context.js

```diff
@@ -10,10 +10,12 @@ class StorageContext {
         this.transaction = transaction
     }
 
-    async getAllUsers() {
+    async getAllUsers(sortByRating = false) {
         let rows
         try {
-            rows = await this.transaction.executeQuery(dbQueries.selectAllusers, [])
+            rows = await this.transaction.executeQuery(
```

Comment:
this is problematic. what if we'll have to have the users sorted by name? or by  registration date? are we intending to add arguments in this function?

sorting users on client will never be problematic, because the number of them will always be super small.

what i suggest:
either getAllUsers().sort(...)
or getUsersSortedByRating()